custom_components/deutschebahn/config_flow.py

```diff
@@ -13,6 +13,8 @@
     CONF_START,
```

Comment:
We will need a solution for our current users. All sensor will get unavailable after updating to this version, since the new attributes are empty for those sensors.

New sensors are getting created fine

A not so gut solution for this would be to include this in a breaking change and alerting the user with the changelog. The user would then have to remove all sensors and add them back in.
We can discuss about that if no better solution is being created.

Reply:
I added an options flow (so bonus feature: reconfiguration!).

I think this should allow for seamless migrations.
But please check my assumption on that one.